content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -41,11 +45,14 @@ some are clearly not very interesting (for example, multiple mesh in a single cl
 
 In a production deployment involving multiple clusters, the deployment may use a
 mix of patterns. For example, having more than one control plane is recommended for HA,
-but you could achieve this for a 3 cluster deployment by deploying 2 clusters with
-a single shared control plane and then adding the third cluster with a second
-control plane in a different network. All three clusters could then be configured
-to share both control planes so that all the clusters have 2 sources of control
-to ensure HA.
+and you can achieve this for a 3 cluster deployment with the following steps:
```

Comment:
re: line 47, is a "pattern" equivalent to dimension?

Reply:
I wrote that sentence a few months ago. By "mix of patterns", I really meant a "mix of answers to the above questions", i.e., cluster 1 and 2 can have one answer for one of the dimensions, while cluster 3 and 4 have a different answer for the same dimension. Maybe "mix of dimensions" or "combination of dimensions" would be a better way to say it?

Reply:
We chose to go with "deployment models" rather than "dimensions" as the title of this doc. We should probably use that here.

Reply:
deployment models or models sgtm.